homeworks/zlinszkygyorgy/3_conditionals_loops/flat_finder.py

```diff
@@ -0,0 +1,22 @@
+# input data (city and rental fee)
+city = input("In which city are you looking for a flat to rent? ").strip().title()
+rental_fee = int(input("What is the rental fee (USD)? "))
+
+# close out Washington
+if city == "Washington":
+    print(f"You cannot rent a flat in {city} for {rental_fee} USD")
+# Chicago for any price
+elif city == "Chicago":
+    print(f"You can rent a flat in {city} for {rental_fee} USD")
+# New York / San Francisco case
+elif city == "New York" or city == "San Francisco":
```

Comment:
Nem hiba, csak egy egyszerűsítési lehetőség:

`elif city in ["New York", "San Francisco"]:`

a membership operátorral rövidíteni tudod a kódot, mert nem kell többször is szembeállítani a változót a vizsgálandó értékkel.